tests/php/test-tag-and-attribute-sanitizer.php

```diff
@@ -1714,6 +1714,26 @@ static function() {
 				[],
 				[ 'invalid_attribute', 'invalid_attribute' ],
 			],
+
+			'illegal_width_attribute'                      => [
+				'<amp-img src="/img1.png" width="50%" height="50"></amp-img>',
+				'',
+			],
+
+			'illegal_height_attribute'                     => [
+				'<amp-img src="/img1.png" width="50" height="50%"></amp-img>',
+				'',
+			],
+
+			'illegal_width_attribute_with_layout'          => [
+				'<amp-img src="/img1.png" width="50" height="50%" layout="responsive"></amp-img>',
+				'',
+			],
+
+			'illegal_height_attribute_with_layout'         => [
+				'<amp-img src="/img1.png" width="50" height="50%" layout="responsive"></amp-img>',
+				'',
+			],
```

Comment:
I suggest adding the following tests, to cover edge cases:
- a value of `0` => `width="0"`
- an empty value => `width=""`
- multiple contradicting values => `width="10" width="20"` (not sure what the result should be... last one used, or invalidated?)